xline/tests/kv_test.rs

```diff
@@ -249,3 +249,55 @@ async fn test_kv_delete() -> Result<(), Box<dyn Error>> {
 
     Ok(())
 }
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 10)]
```

Comment:
Why should we set `worker_threads` as 10? Its default value is the number of cpus on the system, so I guess keeping it as default is better.

Reply:
To unify with other tests, now I have unified all the tests without set `worker_threads`